lib/global/data/path_data.dart

```diff
@@ -0,0 +1,24 @@
+const String PATH_MANIFEST = 'android/app/src/main/AndroidManifest.xml';
+const String PATH_MANIFEST_DEBUG = 'android/app/src/debug/AndroidManifest.xml';
+const String PATH_MANIFEST_PROFILE =
+    'android/app/src/profile/AndroidManifest.xml';
+
+const String PATH_ACTIVITY = 'android/app/src/main/';
+
+const String PATH_APP_LEVEL_GRADLE = 'android/app/build.gradle';
+const String PATH_TO_PLIST = 'ios/Runner/Info.plist';
+const String PATH_TO_PODFILE = 'ios/Podfile';
+const String PATH_AD_UNIT_ID = 'lib/ad_unit_ids/ad_unit_id.dart';
+const String PATH_MAIN = 'lib/main.dart';
+const String PATH_GRADLE_PROPERTIES = 'android/gradle.properties';
+
+const String STRING_PODFILE_IMPORT_GOOGLE_ADS =
+    """pod 'Google-Mobile-Ads-SDK'""";
+const String STRING_PODFILE_IMPORT_APPLOVIN =
+    """pod 'GoogleMobileAdsMediationAppLovin'""";
+
```

Comment:
move these constants to sdk_imports.dart (create new file)

Reply:
these must be in the code string file as these are code lines that are being written.